core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -576,27 +576,26 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
         |  --kill SUBMISSION_ID        If given, kills the driver specified.
         |  --status SUBMISSION_ID      If given, requests the status of the driver specified.
         |
-        | Spark standalone and Mesos only:
+        | Spark standalone, Mesos and K8S only:
```

Comment:
Shell we use fully qualified name since it's a user facing doc?

Reply:
Agree, use "Kubernetes"